### PR TITLE
fix: Refactor utilizable functions

### PR DESCRIPTION
**Pull request by lizable**
This PR refactors webui components by assembling frequently used and duplicated functions into `backendai-common-utils`.
After merging this PR, I would like to suggest both contributors and maintainers regard `backendai-common-utils` as the "common" component of webui project, and using it.

### CODE REVIEW

File: src/components/backend-ai-storage-list.ts [hunk 1]
```diff
@@ -2572,7 +2552,7 @@ export default class BackendAiStorageList extends BackendAIPage {
    * @return {string} UTC time string
    * */
   _humanReadableTime(d) {
-    const date = new Date(d * 1000);
```
Comment (Sujin-Kim1) [inline src/components/backend-ai-storage-list.ts:2575]: parameter `d` is unused.

File: src/components/backend-ai-storage-list.ts [hunk 2]
```diff
@@ -2572,7 +2552,7 @@ export default class BackendAiStorageList extends BackendAIPage {
...
+    const date = new Date();
```
Comment (agatha197) [inline src/components/backend-ai-storage-list.ts]: parameter `d` is unused.

**Comment by inureyes** [comment]
We will drop this branch since it makes components bound each other too strong. Before close, we need to revert flash-blade integration source code.

**Comment by inureyes** [comment]
If some functions / methods are used frequently, please add it to backend-ai-page.

**Pull request closed without merging**